### ISSUE TITLE
NY: No Assembly roll calls scraped this session.

### ISSUE THREAD

**Issue by estaub** [openstates/openstates#1831]
AFAICT we've scraped **no** NY Assembly roll calls for this session.  The state _is_ publishing them, e.g. http://assembly.state.ny.us/leg/?default_fld=&leg_video=&bn=A03021&term=2017&Summary=Y&Floor%26nbspVotes=Y

Related, but probably different: https://github.com/openstates/openstates/issues/1384

**Comment by estaub** [comment]
In the case of the vote referenced above, the API doesn't show the vote at all.  

I noted one unusual condition: the bill, and many other NY bills, has different IDs for the Senate and the Assembly.  The API notes this with the undocumented:

```json
"companions": [{"bill_id": "A 3021", "chamber": "upper", "session": "2017-2018"}]
```
but the Assembly roll call doesn't appear in either the Senate or Assembly bill records.

**Comment by jamesturk** [comment]
after tonight's NY run, check this and #1384

**Comment by jamesturk** [comment]
will also need to re-run against 2017-2018 ideally

**Comment by schneidy** [comment]
@jamesturk Should be fixed as it ties in with issue #1384

**Comment by jamesturk** [comment]
ran for 2017-2018 👍
